etcdmgmt/embed.go

```diff
@@ -0,0 +1,70 @@
+package etcdmgmt
+
+import (
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/coreos/etcd/embed"
```

Comment:
The format I like to follow for imports is to have stdlib packages grouped together, followed by the project packages and then all external packages. Makes it easier to identify packages, and is consistent with the rest of the code.